.github/workflows/vb_script.py

```diff
@@ -36,13 +36,15 @@ def bump_version(version_line, pre_release):
     curr_version = data[-1]
```

Comment:
Perhaps it can be helpful to include an example in the docstring for what this function does in the pre-release and post-release cases.
```
Assume that we have PennyLane version 0.18.0 as the latest release on PyPi.

Pre-release:

We gather the version number for the latest of PennyLane release (i.e., v0.18.0) and increment the middle value to obtain the version of the plugin that we'll release: v0.19.0.

Post-release:

We have just released the v0.19.0 version of the plugin and have v0.19.0 set as the version number. We are entering a new development phase of the plugin attached to the next release, so we increment the version number (0.20.0) and add the "-dev" suffix (0.20.0-dev).
```

Reply:
Got it